app/scripts/controllers/reviewRegistration.js

```diff
@@ -280,4 +280,141 @@ angular.module('confRegistrationWebApp')
     $scope.hasPendingCheckPayment = function(payments){
       return _.some(payments, { paymentType: 'CHECK', status: 'PENDING' });
     };
+
+    ////// START EVENT-433 //////
+
+    // Get spouse registration
+    // End point: getSpouseRegistration = /conferences/conferenceId/registrations/spouse
+    ///// TEST DATA change end point url below to: /conferences/conferenceId/registrations/current /////
+    $http.get('conferences/' + conference.id + '/registrations/spouse')
+      .then(function(response) {
+        if (response.status === 200) {
+          $scope.spouseRegistration = response.data;
+
+          ///// TEST DATA /////
+          // $scope.spouseRegistration.completed = true;
+          // $scope.spouseRegistration.registrants[0].email = 'janeworner3@hotmail.com';
+          // $scope.spouseRegistration.registrants[0].firstName = 'Jane';
+          // $scope.spouseRegistration.registrants[0].lastName = 'Worner3';
+          ///// TEST DATA ////
+
+        }
+        else {
+          $scope.spouseRegistration = {};
+          console.log('No current and/or spouse registration found for user with ID.');
+        }
+      }).catch(function(response) {
+      $scope.spouseRegistration = {};
+      console.log('No current and/or spouse registration found for user with ID. Status = ' + response.status);
+      console.log(response.data);
+    })
+      .finally(function() {
+        checkSpouseRegistration();
+      });
+
+    //Parameters needed to retrieve all current registrants for conference.id
+    var queryParameters = {
+      block: [],
+      //page: 1,
+      //limit: 20,
+      orderBy: 'last_name',
+      order: 'ASC',
+      filter: '',
+      filterPayment: '',
+      filterRegType: '',
+      includeCheckedin: 'yes',
+      includeWithdrawn: 'yes',
+      includeIncomplete: 'yes'
+    };
+
+    //Get all current registrants based on conference.id and queryParameters
+    //then check if spouse is registered and display 'Spouse Registration' section if needed.
+    var checkSpouseRegistration = function() { // jshint ignore:line
+      $http.get('conferences/' + conference.id + '/registrations', {params: queryParameters})
```

Comment:
I did not notice this during our review, but this HTTP call is unnecessary; we aren't even using its response! You can just replace the contents of checkSpouseRegistration with the contents of the `.then` handler, i.e. the code that sets `spouseIncludedOnCurrentRegistration` and `$scope.spouseSectionVisible`.